test/fixtures/makeCompilerWithNullPrototypeOutputFileSystem.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const webpack = require('webpack');
+
+function makeCompiler( config )
+{
+  const compiler = webpack( config );
+
+  const outputFileSystem = Object.create(null);
+
+  compiler.outputFileSystem = outputFileSystem;
+
+  return compiler;
+}
+
+module.exports = makeCompiler;
```

Comment:
I don't think we need another function for this. We only need this behavior in one test case so lets just do it in there.

Reply:
removed